src/scimlfunctions.jl

```diff
@@ -322,6 +343,21 @@ end
 (f::DDEFunction)(args...) = f.f(args...)
 (f::DDEFunction)(::Type{Val{:analytic}},args...) = f.analytic(args...)
 
+function (f::DynamicalDDEFunction)(u,h,p,t)
+  ArrayPartition(f.f1(u.x[1],u.x[2],h,p,t),f.f2(u.x[1],u.x[2],h,p,t))
+end
+function (f::DynamicalDDEFunction)(du,u,h,p,t)
+  f.f1(du.x[1],u.x[1],u.x[2],h,p,t)
+  f.f2(du.x[2],u.x[1],u.x[2],h,p,t)
+end
+function Base.getproperty(f::DynamicalDDEFunction, name::Symbol)
+  if name === :f
+    # Use the f property as an alias for calling the function itself, so DynamicalDDEFunction fits the same interface as DDEFunction as expected by the ODEFunctionWrapper in DelayDiffEq.jl.
```

Comment:
See https://github.com/SciML/DelayDiffEq.jl/pull/203: this is so that `ODEFunctionWrapper` can expect all `AbstractDDEFunction`s to define `f.f` with the entire DDE.